runelite-client/src/main/java/net/runelite/client/ui/FontManager.java

```diff
@@ -93,4 +112,14 @@ public static Font getRunescapeBoldFont()
 	{
 		return runescapeBoldFont;
 	}
+
+	public static LinkedHashMap<Font, String> getFontMap()
```

Comment:
You probably should not expose entire font map

Reply:
Well you are using it just to map font to string, so I guess getFontName(Font) or something like that

Reply:
Yes i think that might work i'll get on that.